src/components/SnackbarManager.tsx

```diff
@@ -48,12 +49,7 @@ const SnackbarManager = ({ message }: { message: SnackbarMessage | undefined })
     <SnackbarManagerContainer>
       <AnimatePresence>
         {message && (
-          <SnackbarPopup
-            initial={{ y: 10, opacity: 0 }}
-            animate={{ y: 0, opacity: 1 }}
-            exit={{ opacity: 0 }}
-            className={message?.type}
-          >
+          <SnackbarPopup {...fadeInBottom} {...fadeOut} className={message?.type}>
```

Comment:
NIT: We use 2 technics to compose animation props:
1) You merge animation objects in `src/animations/index`, in this case this would lead to `fadeInOutBottom`;
2) You use multiple spreads directly in the component props, like here.

We should probably use only one technique. Either we add all combinations used in the app in `src/animations/index`, or we remove from the file all merged animations (`fadeInOut`, `fadeInSlowly`, `fadeOutFast` etc.).

I think that I'm in favour of the first option, in order to keep our components props cleaner.

WDYT?

Reply:
> You merge animation objects in src/animations/index, in this case this would lead to fadeInOutBottom;

Actually, it wouldn't. It should fade in from bottom, but it shouldn't fade out from bottom, it should just fade out. So, technically, it should be `fadeInBottomFadeOut`... and it's not really reused anywhere else.

> We should probably use only one technique.

I personally like that we have the option of composability in JSX. Forcing one technic might lead us to having ugly names like the one I mentioned above... I think we should keep reusable prop combinations in the animations file, and for special cases like this one use composability. WDYT?

Reply:
It was a nit, I'm okay with starting this way and adjusting course if necessary! 🆗